### DISCUSSION TITLE
Is it possible to use Firebase Crashlytics on Windows?

### DISCUSSION THREAD

**Discussion by rasitayaz** [firebase/flutterfire#8766] [Q&A] [answered]
I'm not sure if FlutterFire is (or will be) available for Windows but I wanted to ask if there is a way to use any of the services (especially crashlytics) in a Flutter Windows application.

**Comment by razfazz** [comment]
any news on this?

**Comment by Yevgny** [comment]
Still nothing here? Would be even great if there gonna be at least simple API calls to track issues for Crashlytics for the Flutter Windows.

**Comment by symbol-tabios** [comment]
Another one of the hopeful devs who wanted crashlytics on flutter windows to be supported

**Comment by tranhuudang** [comment]
Yes, seeing how the app crash without a clue of why is frustrating.

**Comment by MichaelVerdon** [comment, marked as answer]
Hi there, windows does not currently support Crashlytics. Here are the list of supported features on Windows Firebase [here](https://firebase.google.com/docs/cpp/setup?platform=ios#libraries-desktop) because Crashlytics is not supported we cannot do the same as it is an upstream issue. Please comment [here](https://firebase.uservoice.com/forums/948424-general) if you want to see this pushed 😄 .
TLDR: Underlying C++ SDK does not support it so it needs to be supported there before we can support it here.
